src/ReinforcementLearningZoo/Manifest.toml

```diff
@@ -456,7 +456,7 @@ version = "0.9.6"
 deps = ["AbstractTrees", "Adapt", "CUDA", "CircularArrayBuffers", "Compat", "Dates", "Distributions", "ElasticArrays", "FillArrays", "Flux", "Functors", "GPUArrays", "LinearAlgebra", "MacroTools", "Markdown", "ProgressMeter", "Random", "ReinforcementLearningBase", "Setfield", "Statistics", "StatsBase", "UnicodePlots", "Zygote"]
 path = "../ReinforcementLearningCore"
 uuid = "de1b191a-4ae0-4afa-a27b-92d07f46b2d6"
-version = "0.8.1"
+version = "0.8.2"
```

Comment:
Did you manually edit this file?
You only need to update this line  https://github.com/JuliaReinforcementLearning/ReinforcementLearning.jl/blob/master/src/ReinforcementLearningZoo/Project.toml#L37

and set it to `0.8.2` instead of `0.8`. Then run `]resolve` and this file will be automatically updated.

Reply:
No, I only change the RLcore version and run `]resolve`.

Reply:
I see. Please update the compat of `RLCore` as I said above, so that it will not accidentally rely on an old version of `RLCore`.

Reply:
Well. I will update it.